dbms/src/AggregateFunctions/AggregateFunctionSumMap.cpp

```diff
@@ -52,23 +52,33 @@ using SumMapArgs = std::pair<DataTypePtr, DataTypes>;
 
 SumMapArgs parseArguments(const std::string & name, const DataTypes & arguments)
 {
-    if (arguments.size() < 2)
-        throw Exception("Aggregate function " + name + " requires at least two arguments of Array type.",
+    DataTypes &args = const_cast<DataTypes &>(arguments);
```

Comment:
That's bad practice - the user doesn't expect that we will modify the argument.

Reply:
fixed